<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileEditWindow</class>
 <widget class="QMainWindow" name="ProfileEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование профиля</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/logo.ico</normaloff>:/resources/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: white;
	font-family: Rubik;
	background-color: #181818
}

QLineEdit {
	border: 2px solid #cfcfcf;
	border-radius: 4px;
}

QPushButton {
    background-color: transparent;
	border: none;
 	border-collapse: separate;
	border-radius: 5px;
}

QPushButton:hover {
    background-color: #999;
}

QPushButton:pressed {
    background-color: #666;
}

QListWidget {
	border: 2px solid #cfcfcf;
	border-radius: 7px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>6.</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group1" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string1">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown1">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group6" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string6">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown6">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="createProfileButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 6px;</string>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group4" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string4">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown4">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group8" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string8">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown8">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>4.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(220, 220, 220)
}</string>
    </property>
    <property name="text">
     <string>Задержка (ms)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group9" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string9">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown9">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group7" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string7">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown7">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>7.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>9.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(220, 220, 220)
}</string>
    </property>
    <property name="text">
     <string>Текст</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>8.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>5.</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group5" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string5">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown5">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group2" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string2">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown2">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QHBoxLayout" name="group3" stretch="4,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="string3">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cooldown3">
       <property name="font">
        <font>
         <family>Rubik</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="shortcutlabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(220, 220, 220)
}</string>
    </property>
    <property name="text">
     <string>Горячая клавиша</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="shortcut">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="keySequence">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="profileName">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="profiileNameLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>341</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(220, 220, 220)
}</string>
    </property>
    <property name="text">
     <string> Название профиля</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffdbdc;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fffde0;</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(220, 220, 220)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Редактор профиля&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
